packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -5225,6 +5253,45 @@ void main() {
       expect(tester.getBottomLeft(find.text(kHelper1)), const Offset(12.0, 76.0));
     });
 
+    testWidgets('InputDecorator shows helper text', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        buildInputDecoratorM2(
```

Comment:
@QuncCccccc do you have any input on whether we should be adding anymore m2 specific tests, or should they all default to m3?

Reply:
I think if this property can be used in M2, then we probably should add these unit tests:) Eventually, these M2-related tests will be moved to the `LegacyThemeData` package.

Reply:
That makes sense. Thank you!

Reply:
Sounds good! Thanks for clarifying.